team/index.html

```diff
@@ -12,10 +12,14 @@
       Core Team, and the Steering Council. Their roles are defined by the
       <a href="https://crystal-lang.org/reference/governance.html">Crystal Governance document</a>.
     </p>
+
+    <p>
+      If you are an active contributor of the project and feel you can level up your involvement with it, you can apply to be a <a href="#core">core member</a> or to have the new <a href="#triage">Triage role</a>. If you want to apply or have questions, send us an email to <a href="mailto:crystal@manas.tech">crystal@manas.tech</a> with your desired role and GitHub handle.  The application process might take up to a month.
```

Comment:
```suggestion
      If you are an active contributor of the project and feel you can level up your involvement with it, you can apply to be a <a href="#core">core member</a> or to have the <a href="#triage">Triage role</a>. If you want to apply or have questions, send us an email to <a href="mailto:crystal@manas.tech">crystal@manas.tech</a> with your desired role and GitHub handle.  The application process might take up to a month.
```

Referring to something as "new" can get old pretty fast IME, I don't think it adds value here :D